src/app/config/projectProperties.json

```diff
@@ -1,50 +1,67 @@
 {
     "projectTypes": {
-        "angular": {
-            "displayname": "Office Add-in project using Angular framework",
+        "taskpane": {
+            "displayname": "Office Add-in Task Pane project",
```

Comment:
I notice that we've removed "using JQuery framework" from the end of the name for this project type. Although I understand that this vanilla/basic project type is not technically a `JQuery` project, I propose that we still need some way to differentiate this project type from the other 2 task pane project types (i.e., "...using Angular framework" and "...using React framework").  Technically, each one of the first 3 project types in the list is a "Office Add-in Task Pane project" -- so we should name this first project type in a way that differentiates it from the next two types (Angular and React). 

@JuaneloJuanelo any suggestions for what this basic/vanilla project type should be called?

Reply:
I like "Office Add-in Task Pane". The others are "Office Add-in Task Pane using Angular" or "...using React".

Reply:
Upon further consideration, I think it's fine as is ("Office Add-in Task Pane project").

Reply:
I will update the string when I get into the office

Reply:
I agree with Kim. Let's leave as is